job_templates/packaging_job.xml.em

```diff
@@ -134,7 +134,7 @@ if [ -n "${CI_AMENT_TEST_ARGS+x}" ]; then
   if [[ "$CI_AMENT_TEST_ARGS" != *-- ]]; then
     CI_AMENT_TEST_ARGS="$CI_AMENT_TEST_ARGS --"
   fi
-  export CI_ARGS="$CI_ARGS --ament-test-args $CI_AMENT_TEST_ARGS"
+  export CI_ARGS="$CI_ARGS --ament-test-args $CI_AMENT_TEST_ARGS --"
```

Comment:
l134 adds the delimiters beforehand, not afterwards.
Without the delimiter added at the end, it fails with:
```
"/home/rosbuild/ci_scripts/venv/bin/python" -u ""./src/ament/ament_tools/scripts/ament.py"" test "src" --build-space "build" --install-space "install" --only-packages ros1_bridge --skip-build --skip-install --retest-until-pass 5 --ros1-path /opt/ros/kinetic
02:31:45 usage: ament.py [-h]
02:31:45                 [build | build_pkg | list_packages | package_name | package_version | test | test_pkg | test_results | uninstall | uninstall_pkg]
02:31:45                 ...
02:31:45 ament.py: error: unrecognized arguments: --ros1-path /opt/ros/kinetic
```

I believe because `--ros1-path` is consumed as part of the test args group [here](https://github.com/ros2/ci/blob/c1e34ffc70da8b0ead7ee94d03eae52225f3a265/ros2_batch_job/__main__.py#L150)